app/javascript/components/cloud-volume-form/cloud-volume-form.schema.js

```diff
@@ -0,0 +1,55 @@
+import { componentTypes, validatorTypes} from '@@ddf';
+import { parseCondition } from '@data-driven-forms/react-form-renderer';
+
+const createSchema = (fields, promise, edit, loadSchema) => {
+  const idx = fields.findIndex((field) => field.name === 'volume_type');
+  const loadPromise = () =>  {
+    const supports = edit ? 'supports_cloud_volume' : 'supports_cloud_volume_create';
+    return API.get(`/api/providers?expand=resources&attributes=id,name,${supports}&filter[]=${supports}=true`);
+  };
+
+  return ({
+    fields: [
+      {
+        component: componentTypes.SELECT,
+        name: 'ems_id',
+        id: 'ems_id',
+        label: __('Storage Manager'),
+        placeholder: __('<Choose>'),
+        onChange: promise.then(loadSchema()),
+        loadOptions: () => loadPromise.then(({ resources }) => resources.map(({ id, name }) => ({ value: id, label: __(name) }))),
```

Comment:
The `loadPromise` here is a function definition, on which you're calling a promise handler that will obviously fail. You probably wanted to write `loadPromise().then`, but IMO you should not have half of the handler here and the other half in a named function. It's up to you if you want to keep it here or move the whole thing into the function definition above. But if you chose the second option, please rename it to `loadOptions` for consistency...